flake.lock

```diff
@@ -33,11 +33,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1658311025,
-        "narHash": "sha256-GqagY5YmaZB3YaO41kKcQhe5RcpS83wnsW8iCu5Znqo=",
+        "lastModified": 1658644204,
+        "narHash": "sha256-MWyfCH9K3eVTXJUxBi67OQSAh9jJAnvWklM6qm4j8w8=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "cd8d1784506a7c7eb0796772b73437e0b82fad57",
+        "rev": "2f0c3be57c348f4cfd8820f2d189e29a685d9c41",
```

Comment:
Just curious, was there a specific change in nixpkgs that we need here?

Reply:
Just a routine `nix flake update` by habit, but will revert, I'm not aware of a needed change.